webapp/src/store/index.js

```diff
@@ -26,6 +27,40 @@ export default new Vuex.Store({
 			return (permission) => {
 				return !!state.permissions?.includes(permission)
 			}
+		},
+		flatSchedule (state) {
+			if (!state.schedule) return
+			const sessions = []
+			for (const day of state.schedule.schedule) {
+				for (const room of day.rooms) {
+					const vRoom = state.rooms.find(r => r.import_id === state.world.pretalx.room_mapping[room.id])
```

Comment:
Heads up, I changed how room mappings work on master. Should now be less fragile and easier to work with